### ISSUE TITLE
Risk of Rain 2 Patch 1.2.2.0 (March 11th, 2022) Broke exiting drop pod in Multiplayer sessions for non-hosts.

### ISSUE THREAD

**Issue by Yangjo** [DrBibop/RoR2VRMod#62]
Perfectly able to exit a Single Player Drop Pod or as Host, but unable to do so if I join another player. 

EDIT: Seems to be affected by another mod, attempting to figure out.

**Comment by DrBibop** [comment]
Sounds good. Let me know how it goes.

**Comment by Yangjo** [comment]
> Sounds good. Let me know how it goes.

Seemed like there issue was related to "BetterUnityPlugin"
https://thunderstore.io/package/XoXFaby/BetterUnityPlugin/ 

When I disabled this mod, I was able to properly exit the Drop Pod. Also noticeably, this BetterUnityPlugin affected the initial "center the HUD" at the main menu screen. I would have issues registering inputs there, with this mod enabled. It's a very minor mod, as this mod was required for the "UnlockAll" mod. https://thunderstore.io/package/XoXFaby/UnlockAll/

Seems very minor, as I only used this mod for unlocking any Modded characters, which I would be unlikely using while playing VR.
